README.md

```diff
@@ -120,8 +121,13 @@ If you find an issue with this package and have a fix, please feel free to open
 
 ## Testing
 
-If you contribute to this package you can run the tests using `yarn test`. This package also includes a sample application for manual testing. First open a seperate terminal and run `yarn watch`, this will ensure the Typescript files in the src directory are compiled to Javascript in the lib directory. Navigate to `src/sample`, here you can edit `index.ts` to your liking and test your contributions manually. Run `npx cdk --app lib/sample/index.js <CDK Command>`, replacing `<CDK Command>` with common CDK commands like `synth`, `diff`, or `deploy`. 
-* Note, if you receive "... is not authorized to perform: ..." you may also need to authorize the commands with your AWS credentials.
+If you contribute to this package you can run the tests using `yarn test`. This package also includes a sample application for manual testing. First open a seperate terminal and run `yarn watch`, this will ensure the Typescript files in the src directory are compiled to Javascript in the lib directory. Navigate to `src/sample`, here you can edit `index.ts` to your liking and test your contributions manually. Run `npx cdk --app lib/sample/index.js <CDK Command>`, replacing `<CDK Command>` with common CDK commands like `synth`, `diff`, or `deploy`.
+
+- Note, if you receive "... is not authorized to perform: ..." you may also need to authorize the commands with your AWS credentials.
+
+### Debug Logs
+
+To display the debug logs for this library, set the `DD_CONSTRUCT_DEBUG_LOGS` env var to `true` when running `cdk synth`.
```

Comment:
Right now it isn't completely clear how to enable the env var, does it get enabled in the actual cdk synth command? Can you provide a snippet of the command?